pkgs/development/compilers/zig/0.13/hook.nix

```diff
@@ -18,6 +18,7 @@ makeSetupHook {
     zig_default_flags = [
       "-Dcpu=baseline"
       "--release=safe"
+      "--color off" # Turn off progress output
```

Comment:
Is this a new feature in 0.13? There's a generic hook in https://github.com/jmbaur/nixpkgs/blob/zig-0.13-build-noprogress/pkgs/development/compilers/zig/hook.nix

Reply:
Yeah this progress output is specific to 0.13, it looks like this when running zig build interactively: https://asciinema.org/a/661404, but doesn't print well when building with nix.